packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.1.1
+
+- Removed assertion `too many routes for location`
+- Fixes (https://github.com/flutter/flutter/issues/99833)
```

Comment:
These two are actually the same ? they should be merged into one bulletin.
maybe
```
- Uses first match if there is more than one route to match. [ [#99833](https://github.com/flutter/flutter/issues/99833)
```

Reply:
Fixed